Source/CMakeLists.txt

```diff
@@ -82,6 +82,9 @@ set(libdevilutionx_SRCS
   controls/modifier_hints.cpp
   controls/plrctrls.cpp
   controls/keymapper.cpp
+  controls/touch/event_handlers.cpp

```

Comment:
`if(NOT USE_SDL1)`?

Reply:
I went with a in-file guard instead since that is also what we do for `game_controller.cpp` and some other files.